SignalServiceKit/src/Storage/OWSStorage.m

```diff
@@ -234,7 +234,11 @@ - (nullable Class)unarchiver:(NSKeyedUnarchiver *)unarchiver
     cannotDecodeObjectOfClassName:(NSString *)name
                   originalClasses:(NSArray<NSString *> *)classNames
 {
-    OWSProdLogAndFail(@"%@ Could not decode object: %@", self.logTag, name);
+    if ([name isEqualToString:@"TSRecipient"]) {
+        DDLogError(@"%@ Could not decode object: %@", self.logTag, name);
```

Comment:
Removed `fail` for known missing types so that we can test them on debug devices and verify we're handling them properly.

Reply:
👍